### PR TITLE
Sitzmann2020implicit Summary 1

### PR DESCRIPTION
**Pull request by rll2396**

### CODE REVIEW

File: _summaries/sitzmann2020implicit_1.md [hunk 1]
```diff
@@ -1,19 +1,76 @@
...
+        * the SDF values are negative when outside the shape and positive when inside the shape
+        * gradient is $$1$$ almost everywhere
+        * distance values on the shape should be close to 0
+        * distance values off the shape should not be close to 0
+        * for calculating loss, sample random points in the domain, half on the shape half off it
+
+    ![Helmholtz](sitzmann2020implicit_1e.png)
+
+    * Helmholtz equation
+        * Supervise with loss function based on helmholtz equation, which relates the Laplacian of a function with itself
```
Comment (aabayomi) [inline _summaries/sitzmann2020implicit_1.md:63]: I think adding more details on what the experiment is about, what data they used, network architecture and maybe hyper-parameter used will be good.

File: _summaries/sitzmann2020implicit_1.md [hunk 2]
```diff
@@ -1,19 +1,76 @@
...
+
+    ![CelebA](sitzmann2020implicit_1d.png)
+
+    * Learning a space of implicit functions
```
Comment (aabayomi) [inline _summaries/sitzmann2020implicit_1.md:67]: I think this was the only experiment the authors tried out any form of training. you can add the table showing the pixel-wise MSE with maybe a little more details.

File: _summaries/sitzmann2020implicit_1.md [hunk 3]
```diff
@@ -1,19 +1,76 @@
...
+        * the resulting SIREN can fill in the rest of the image
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Sine activation functions are effective at modeling implicit problems
+* Requires careful initialization
+* Demonstratably achieves much better performance on a variety of tasks
```
Comment (aabayomi) [inline _summaries/sitzmann2020implicit_1.md:76]: You may add, sine / periodic activation functions are good for describing complex signals and their derivatives.

**Review by aabayomi — COMMENTED** [review]
The summary is original and easy to follow.

File: _summaries/sitzmann2020implicit_1.md [hunk 4]
```diff
@@ -1,19 +1,76 @@
...
+        * SIREN is used to solve the Poisson equation
+        * goal is to reconstruct an image
+        * the gradient or Laplacian of the ground truth image is used for supervision
+    * Poisson image editing
+        * SIREN is used to create an image from the linear interpolation of the gradients of two images
+
+    ![SDFs](sitzmann2020implicit_1b.png)
+
+    * Representing shapes with signed distance functions
+        * SDF outputs the distance to the boundary of the shape
```
[the remark this answers is not in the document]
Comment (rll2396) [reply _summaries/sitzmann2020implicit_1.md:53]: Is the context just that a previous work showed good results on shape representation using SDFs?

[the remark this answers is not in the document]
Comment (aabayomi) [reply _summaries/sitzmann2020implicit_1.md:53]: Yes, I think SDFs are usually used in geometric rendering problems so i believe they tried using SIREN to fit the distances.
